warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -1166,108 +1173,141 @@ protected ASTJexlScript processTree(final ASTJexlScript originalQueryTree, Shard
             stopwatch.stop();
         }
         
-        if (!disableBoundedLookup) {
-            stopwatch = timers.newStartedStopwatch("DefaultQueryPlanner - Expand bounded query ranges");
-            
-            // Expand any bounded ranges into a conjunction of discrete terms
+        ParallelIndexExpansion regexExpansion = null;
+        if (!disableBoundedLookup && (planMetadata.hasRegexNodes() || planMetadata.hasAnyFieldNodes())) {
```

Comment:
Wouldn't we need to recalculate the plan metadata just before this since many steps have occurred since its calculation?